gateleen-kafka/pom.xml

```diff
@@ -23,10 +23,13 @@
             <version>${project.version}</version>
         </dependency>
         <dependency>
-            <groupId>log4j</groupId>
-            <artifactId>log4j</artifactId>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-api</artifactId>
```

Comment:
Agree, can be removed from the kafka module. Anyway, we don't need it in the playground, because it uses some runconfig module that then has the dependency to log4j